source/main/physics/Beam.cpp

```diff
@@ -5932,32 +5923,28 @@ bool Beam::LoadTruck(
 							dash->loadDashBoard("default_dashboard7000_mph.layout", false);
 							// TODO: load texture dashboard by default as well
 							dash->loadDashBoard("default_dashboard7000_mph.layout", true);
-						}
-						else
+						} else
 						{
 							dash->loadDashBoard("default_dashboard3500_mph.layout", false);
 							// TODO: load texture dashboard by default as well
 							dash->loadDashBoard("default_dashboard3500_mph.layout", true);
 						}
-					}
-					else
+					} else
 					{
 						if (engine->getMaxRPM() > 3500)
 						{
 							//7000 rpm tachometer thanks to klink
 							dash->loadDashBoard("default_dashboard7000.layout", false);
 							// TODO: load texture dashboard by default as well
 							dash->loadDashBoard("default_dashboard7000.layout", true);
-						}
-						else
+						} else

```

Comment:
The old one, because it's mixed:
```
// okay
if (a)
{
    foo();
}
else
{
    bar();
}
// also okay (Java)
if (a) {
    foo();
} else {
    bar();
}
// the awful mix
if ()
{
    foo();
} else
{
    bar();
}
```